packages/mdc-textfield/_mixins.scss

```diff
@@ -45,7 +49,7 @@
 
     33% {
       animation-timing-function: cubic-bezier(.5, 0, .701732, .495819);
-      transform: translateX(10px) translateY(-#{$positionY}) scale(.75, .75);
+      transform: translateX(5px) translateY(-#{$positionY}) scale(.75, .75);
```

Comment:
Outlined requires a smaller value because otherwise the label would overlap the outline, and I think it's better to keep the values the same if possible. I don't think this is redlined so... 5px for all text fields looks good to me. Thoughts?

Reply:
That's probably fine, I would make a quick screen grab of it in Quicktime to send to Jonas for approval. He is the owner of Text Field animations.